src/frontend/Deprecation_analysis.ml

```diff
@@ -85,50 +88,52 @@ let rec collect_deprecated_expr deprecated_userdefined
     ({expr; emeta} : (typed_expr_meta, fun_kind) expr_with) :
     (Location_span.t * string) list =
   match expr with
-  | GetLP ->
-      acc
-      @ [ ( emeta.loc
-          , "The no-argument function `get_lp()` is deprecated. Use the \
-             no-argument function `target()` instead." ) ]
   | FunApp (StanLib FnPlain, {name= "abs"; _}, [e])
     when Middle.UnsizedType.is_real_type e.emeta.type_ ->
       collect_deprecated_expr deprecated_userdefined
         ( acc
         @ [ ( emeta.loc
             , "Use of the `abs` function with real-valued arguments is \
-               deprecated; use functions `fabs` instead." ) ] )
+               deprecated; use function `fabs` instead." ) ] )
```

Comment:
Isn't `fabs()` going to be deprecated soon? We won't need this `abs()` branch at all.

Reply:
I'm not sure about the status of #475 so I just left this unchanged for now. It should be handled alongside the changes to the supported signatures